### DISCUSSION TITLE
[Feature] ability to retain "description" and Geo location when "replace with upload"

### DISCUSSION THREAD

**Discussion by quitchat** [immich-app/immich#13844] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a picture that I entered description and geo location after uploading to immich. After a few days, I made some post processing to the original image, and "replaced" the existing picture with the newly processed one. However, the "description" and the geo location information were lost. It would be nice to retain such information, because most of the time, when you replace an existing picture, the information would still be relevant.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by quitchat** [comment]
1. "old" picture with Description and Geo Location entered after uploaded to Immich (info not embeded in the picture)
![1](https://github.com/user-attachments/assets/76ca2c60-44e5-4845-8fdf-579115377a6e)
2. "Replace with upload"
![2](https://github.com/user-attachments/assets/bc2c2b97-a7f9-4985-bd2e-b1b79af75eb4)
3. Info lost... would be nice if able to retain...
![3](https://github.com/user-attachments/assets/1aae856d-b42f-444d-ab14-05461643613e)

**Comment by bo0tzz** [comment]
Closing in favour of #14356; which is technically newer but has more discussion.
